modules/openapi-generator/src/test/resources/3_1/kotlin/multiplefiles_optinal_okhttp.yaml

```diff
@@ -0,0 +1,24 @@
+openapi: 3.1.0
+info:
+  title: XXXXXXX
+  version: latest
+paths:
+  /multifile/test:
+    post:
+      operationId: operation
+      requestBody:
+        content:
+          multipart/form-data:
+            schema:
+              type: object
+              properties:
+                attachments:
+                  type: array
+                  description: attached files
+                  items:
+                    type: string
+                    format: binary
+                pet_id:
+                  type: string
+                  description: Pet ID
+
```

Comment:
how is this new test spec used in the PR?

did you intend to create a config file under ./bin/config to test with this spec?

Reply:
Sure! I will ping you. As I mentioned, I didn't really get a clear understanding on how to add the tests to check this fix (I tested it against my own project), so some guidance on how to do it would be extremely appreciated!

I added that test spec just by looking at the project, but I didn't know what to do with it.

I'll poke you in Slack some time during the day!